### DISCUSSION TITLE
Statistics

### DISCUSSION THREAD

**Discussion by juliandevelops** [NickRTR-Schule/Vokabeltrainer#21] [Concept & Draft]
Auf dem Dashboard gibt es in der oberen Reihe mehrere Statistikfelder.
Was enthalten diese Felder, bzw. wie viele davon brauchen wir letztendlich auch

**Comment by Adrian-Cxll** [comment]
Ein paar Vorschläge für Widgets:
- durchschnittlicher lernstand insg. : Prozentwert innerhalb Ringchart
- anzahl richtig beantwortete Vokabeln letzter 7 Tage : Zahl innerhalb Ringchart
- zuletzt gelernte Vokabelliste -> lernstand : Prozentwert innerhalb Ringchart
- Vokabeln insg. gelernt : Zahl
- Lernaktivität (z.B. anzahl Vokabeln richtig pro Tag/Woche) : (linienchart/balkenchart)
- Anzahl Vokabeln : Zahl
- Anzahl Vokabellisten : Zahl

**Reply by juliandevelops** [reply]
Kannst du mal Beispiele für ein Ringchart posten? Denkst du mehr an ein wirkliches Ringchart oder eher an ein piechart?

**Comment by juliandevelops** [comment]
Charts sind eine gute Idee, aber wir sollten nicht zu oft die gleiche Art von Charts zu oft benutzen

**Reply by Adrian-Cxll** [reply]
Ich denke mal je nachdem, wie das Layout aussehen wird.
Wenn man z.B. jedes Mal ein anderes Diagramm für Prozentwerte nimmt wäre es mögl. unübersichtlicher?

**Reply by juliandevelops** [reply]
True, müssen wir schauen. Aber halt 3 RoundCharts nebeneinander könnten auch bisschen verwirrend sein.
Probieren wir es einfach aus

**Comment by Adrian-Cxll** [comment]
Dann wäre auch eine Ansicht für alle Stats wichtig.
Je nachdem wie viele Statistikfelder wir übernehmen wird die home page ziemlich voll.

**Reply by juliandevelops** [reply]
Das ist die nächste Frage. Aktuell tendiere ich zu 3 Feldern und keiner Scrollbar
